tests/integration_test.rs

```diff
@@ -642,3 +673,42 @@ async fn test_package_caching(
     assert!(options.pack_options.output_file.exists());
     assert!(output_file2.exists());
 }
+
+#[rstest]
+#[tokio::test]
+async fn test_pypi_sdist_fail(
+    #[with(PathBuf::from("examples/pypi-packages/pixi.toml"))] options: Options,
+) {
+    let pack_result = pixi_pack::pack(options.pack_options).await;
+    assert!(pack_result.is_err());
+    // Error: package pysdl2 is not a wheel file, we currently require all dependencies to be wheels.
+    assert!(pack_result.err().unwrap().to_string().contains("pysdl2"));
+}
+
+#[rstest]
+#[tokio::test]
+async fn test_pypi_experimental_support(
+    #[with(PathBuf::from("examples/pypi-bdist-packages/pixi.toml"))] options: Options,
+    #[with(true)] required_fs_objects: Vec<&'static str>,
+) {
```

Comment:
this is the same as `test_simple_python`. let's parametrize this and rename it to `test_simple`